packages/notion/client.ts

```diff
@@ -32,9 +33,81 @@ export async function getAvaiableElections(
   };
 }
 
+export const getElectionPage = async (pageId: string) => {
+  const { results } = await notion.blocks.children.list({ block_id: pageId });
+
+  const { id: candidateDatabaseId } = results.filter(
+    (block) =>
+      'type' in block &&
+      block.type === 'child_database' &&
+      block?.child_database?.title === 'Partidos',
```

Comment:
Surprisingly... NO! hahahahahaha

Basically if type is in block and type is `child_database` then it will have the title property. But I will extract this logic from here, is really crowded.

Reply:
Enhanced